master/pom.xml

```diff
@@ -34,9 +34,6 @@
     <sakai.ehcache.artifactId>ehcache-core</sakai.ehcache.artifactId>
     <sakai.ehcache.version>2.6.11</sakai.ehcache.version>
     <sakai.elasticsearch.version>1.7.6</sakai.elasticsearch.version>
-    <sakai.hibernate.groupId>org.hibernate</sakai.hibernate.groupId>
-    <sakai.hibernate.artifactId>hibernate-core</sakai.hibernate.artifactId>
-    <sakai.hibernate.version>4.3.11.Final</sakai.hibernate.version>
```

Comment:
Please leave sakai.hibernate.version as its easier to adjust here than changing the dependency.